app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :users, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :posts, class_name: 'Post', foreign_key: 'post_id'
+
+  after_create :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
+end
```

Comment:
- [ ] Please consider the same suggestion here as mentioned above to use singular name for post and user in the belongs_to association.
- [ ] [OPTIONAL] Also please consider using `after_save` for  #update_likes_counter instead of `after_create` here as well and make it a private method.

Example snippet:


```rb

class Like < ApplicationRecord
  belongs_to :user
  belongs_to :post

  after_save :update_likes_counter

private

  def update_likes_counter
    post.increment!(:likes_counter)
  end
end
```